torch/_logging/_internal.py

```diff
@@ -515,6 +516,9 @@ def pad_to(s, length=30):
 
   TORCH_LOGS="+some.random.module,schedule" will set the log level of
   some.random.module to logging.DEBUG and enable the schedule artifact
+
+  TORCH_LOGS_FORMAT="%(levelname)s: %(message)s" or any provided format
```

Comment:
what are all the valid keys here? What about frame counts? timestamps?

Reply:
I can list out all the valid keys